docs/user-guide/global-configurations/manage-notification.md

```diff
@@ -1,138 +1,403 @@
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Only superadmins can manage configurations.
+{% endhint %}
 
-Click `Add` and configure SES.
+This section allows you to perform the configurations required for your notifications to work. Make sure you finish this step before [creating a notification](#create-notifications).
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/manage-notification/gc-notification-configuration-ses-config.jpg)
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/manage-notification/configurations-tab.jpg)
 
-| Key | Description |
-| :--- | :--- |
-| `Configuration Name` | Provide a name to the SES Configuration. |
-| `Access Key ID` | Valid AWS Access Key ID. |
-| `Secret Access Key` | Valid AWS Secret Access Key. |
-| `AWS Region` | Select the AWS Region from the drop-down menu. |
-| `E-mail` | Enter the SES verified e-mail id on which you wish to receive e-mail notifications. |
+Here you can add and manage the following configurations:
+* [SES Configuration](#ses-configuration)
+* [SMTP Configuration](#smtp-configuration)
+* [Slack Configuration](#slack-configuration)
+* [Webhook Configuration](#webhook-configuration)
 
-Click `Save` to save your SES configuration or e-mail ID
+{% hint style="info" %}
+### Tip
+At any point in time, you have the option to edit or delete an existing configuration or add more as per your requirements.
+{% endhint %}
 
+### SES Configuration
 
-### **Manage SMTP Configurations**
+{% hint style="info" %}
+### Prerequisites
+* [Generate SMTP Credentials from AWS SES](https://docs.aws.amazon.com/ses/latest/dg/smtp-credentials.html) - To get the user access key and secret key.
+* [Verify Email Identities in AWS SES](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#verify-email-addresses-procedure) - This is to verify the sender's email address (e.g., *michael@devtron.ai*) you wish to use for sending email notifications.
+{% endhint %}
 
-You can manage the `SMTP configuration` to receive e-mails by entering the valid credentials. Make sure your e-mail is verified by SMTP.
+You can use Amazon Simple Email Service (SES) to send email notifications to the recipients.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/manage-notification/smtp-configurations.jpg)
+1. Click **Add**.
 
-Click `Add` and configure SMTP.
+2. Enter the following details in the **Configure SES** window:
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/manage-notification/configure-smtp.jpg)
+    | Key | Description |
+    | --- | ----------- |
+    | **Configuration Name** | Give a name to your SES Configuration, e.g., `Email SES 2024` |
+    | **Access Key ID** | Valid user ID from your AWS SMTP Settings page, e.g., `AKIAWEAVHF123ABCD123` |
```

Comment:
@badal773 Shouldn't this be aws accesskey and secret instead of AWS smtp creds?

Reply:
yes